common-arcana.lic

```diff
@@ -51,6 +59,29 @@ module DRCA
     ['Your body is willing', 'You have not recovered'].include?(result)
   end
 
+  def start_barb_abilities(skills, settings)
+    skills
+      .each do |name|
+        next if DRSpells.active_spells[name]
+        activate_barb_buff(name, settings.sit_meditations)
+        waitrt?
+        pause 6 if get_data('spells').barb_abilities[name]['type'].eql? 'meditation'
+      end
+  end
+
+  def activate_barb_buff(name, sit = false)
+    ability_data = get_data('spells').barb_abilities[name]
+    is_meditation = ability_data['type'].eql? 'meditation'
+    DRC.bput('sit', 'You sit', 'You are already', 'You rise') if sit && is_meditation && !sitting?
+    case bput(ability_data['start_command'], ability_data['activated_message'], 'But you are already', 'Your inner fire lacks', 'find yourself lacking the inner fire', 'You should stand')
+    when 'You should stand'
```

Comment:
Is there a 'you should sit' case we could handle to negate the need for a meditation-specific setting?

Reply:
Done.